### DISCUSSION TITLE
Is TLS v1.3 implemented

### DISCUSSION THREAD

**Discussion by Mel0man1ac** [xipki/xipki#280] [Q&A]
Hi everyone,

I saw on the description that TLS (RFC 7633) is implemented but can anyone tell me if the TLSv1.3 is also supported ?

Greetings

**Comment by xipki** [comment]
RFc 7633 specifies the certificate extension TLS-feature. It has nothing to do with TLSv1.3 support of XiPKI software during the transport.
